app/src/main/java/com/example/makeitso/MakeItSoMessagingService.kt

```diff
@@ -0,0 +1,18 @@
+package com.example.makeitso
+
+import android.util.Log
+import com.google.firebase.messaging.FirebaseMessagingService
+
+class MakeItSoMessagingService : FirebaseMessagingService() {
```

Comment:
Consider overriding the `onMessageReceived` function to construct a notification - it will ensure that the notification is shown when the app is in the foreground: https://firebase.google.com/docs/cloud-messaging/android/receive#handling_messages

Reply:
Done!